### PR TITLE
Add prereqs section to grants tutorials and update API Identifier des…

### PR DESCRIPTION
**Pull request by khorne3**
https://auth0-docs-content-pr-4825.herokuapp.com/docs/api-auth/tutorials/authorization-code-grant-pkce

https://auth0-docs-content-pr-4825.herokuapp.com/docs/api-auth/tutorials/authorization-code-grant

https://auth0-docs-content-pr-4825.herokuapp.com/docs/api-auth/tutorials/client-credentials

https://auth0-docs-content-pr-4825.herokuapp.com/docs/api-auth/tutorials/implicit-grant

https://auth0-docs-content-pr-4825.herokuapp.com/docs/api-auth/tutorials/password-grant

### CODE REVIEW

File: articles/api-auth/tutorials/authorization-code-grant-pkce.md [hunk 1]
```diff
@@ -12,6 +12,15 @@ This tutorial will help you implement the Authorization Code (PKCE) grant. If yo
 
 The __Authorization Code with PKCE__ is the OAuth 2.0 grant that [native apps](/quickstart/native) use in order to access an API. In this document we will work through the steps needed in order to implement this: create a code verifier and a code challenge, get the user's authorization, get a token and access the API using the token.
 
+::: panel Prerequisites
+
+Before beginning this tutorial, please:
+
+* Set the Client's [Grant Type property](/clients/client-grant-types)
+* [Register the API](/apis#how-to-configure-an-api-in-auth0) with Auth0
+* Set the [Default Audience and/or Default Directory](/dashboard-account-settings#api-authorization-settings)
```
Comment (mpaktiti) [inline articles/api-auth/tutorials/authorization-code-grant-pkce.md]: To my knowledge, the Default Directory is only relevant to the Password Grant

File: articles/api-auth/tutorials/authorization-code-grant.md [hunk 2]
```diff
@@ -12,6 +12,15 @@ This tutorial will help you implement the Authorization Code grant. If you are l
 
 The __Authorization Code__ is an OAuth 2.0 grant that [regular web apps](/quickstart/webapp) use in order to access an API. In this document we will work through the steps needed in order to implement this: get the user's authorization, get a token and access the API using the token.
 
+::: panel Prerequisites
+
+Before beginning this tutorial, please:
+
+* Set the Client's [Grant Type property](/clients/client-grant-types)
+* [Register the API](/apis#how-to-configure-an-api-in-auth0) with Auth0
+* Set the [Default Audience and/or Default Directory](/dashboard-account-settings#api-authorization-settings)
```
Comment (mpaktiti) [inline articles/api-auth/tutorials/authorization-code-grant.md]: To my knowledge, the Default Directory is only relevant to the Password Grant

File: articles/api-auth/tutorials/client-credentials.md [hunk 3]
```diff
@@ -5,14 +5,14 @@ description: How to request tokens for your applications.
...
-To start using the Client Credentials Grant, the following must apply:
-- You have configured a Non Interactive Client at the Dashboard, for your application.
-- You have configured an API at the Dashboard, for your API.
-- You have authorized the Non Interactive Client to access your API.
+Before beginning this tutorial, please:
 
-You can do this either [using the Dashboard](/api-auth/config/using-the-auth0-dashboard) or [using the Management API](/api-auth/config/using-the-management-api).
+* Set the Client's [Grant Type property](/clients/client-grant-types)
+* [Register the API](/apis#how-to-configure-an-api-in-auth0) with Auth0
+* Set the [Default Audience and/or Default Directory](/dashboard-account-settings#api-authorization-settings)
```
Comment (mpaktiti) [inline articles/api-auth/tutorials/client-credentials.md]: To my knowledge, the Default Directory is only relevant to the Password Grant

File: articles/api-auth/tutorials/implicit-grant.md [hunk 4]
```diff
@@ -13,6 +13,15 @@ This tutorial will help you implement the Implicit Grant. If you are looking for
 
 The __Implicit Grant__ is an OAuth 2.0 flow that [client-side apps](/quickstart/spa) use in order to access an API. In this document we will work through the steps needed in order to implement this: get the user's authorization, get a token and access an API using the token.
 
+::: panel Prerequisites
+
+Before beginning this tutorial, please:
+
+* Set the Client's [Grant Type property](/clients/client-grant-types)
+* [Register the API](/apis#how-to-configure-an-api-in-auth0) with Auth0
+* Set the [Default Audience and/or Default Directory](/dashboard-account-settings#api-authorization-settings)
```
Comment (mpaktiti) [inline articles/api-auth/tutorials/implicit-grant.md]: To my knowledge, the Default Directory is only relevant to the Password Grant

File: articles/api-auth/tutorials/authorization-code-grant-pkce.md (hunk 1 above)
Comment (mpaktiti) [reply articles/api-auth/tutorials/authorization-code-grant-pkce.md]: I'm wondering whether this last bullet should be there (and all other docs in this PR). It's not actually a prerequisite, it's something that the users can do optionally. You could convert this panel to a paragraph and add a note saying that optionally they can set this and that.

File: articles/api-auth/tutorials/authorization-code-grant-pkce.md [hunk 5]
```diff
@@ -12,6 +12,15 @@ This tutorial will help you implement the Authorization Code (PKCE) grant. If yo
 
 The __Authorization Code with PKCE__ is the OAuth 2.0 grant that [native apps](/quickstart/native) use in order to access an API. In this document we will work through the steps needed in order to implement this: create a code verifier and a code challenge, get the user's authorization, get a token and access the API using the token.
 
+::: panel Prerequisites
+
+Before beginning this tutorial, please:
+
+* Set the Client's [Grant Type property](/clients/client-grant-types)
```
Comment (mpaktiti) [inline articles/api-auth/tutorials/authorization-code-grant-pkce.md]: I'm not sure I get this first prerequisite. Yes, depending on the user's case, they might have to enable or disable grant types, but is it actually a prerequisite to do so? The only must-do that I see is create a client and an API.

**Review by mpaktiti — CHANGES_REQUESTED** [review]
Hey Katie, left some comments, let me know what you think

File: articles/api-auth/tutorials/authorization-code-grant-pkce.md (hunk 5 above)
Comment (khorne3) [reply articles/api-auth/tutorials/authorization-code-grant-pkce.md]: I think the goal is to have all clients have a grant type. You're right in that it's not a must-do, so I changed it to a "check to see if..." kind of thing

**Review by mpaktiti — APPROVED** [review]
Looks good Katie 👍

**Pull request merged**